src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs

```diff
@@ -88,41 +83,33 @@ public void WalletCanReorg()
             // Mine the second transaction back in to the main chain
             using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-                CoreNode stratisReceiver = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-                CoreNode stratisReorg = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-
-                builder.StartAll();
+                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
+                CoreNode stratisReceiver = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
+                CoreNode stratisReorg = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
 
                 int maturity = (int)stratisSender.FullNode.Network.Consensus.CoinbaseMaturity;
                 TestHelper.MineBlocks(stratisSender, maturity + 15);
 
                 int currentBestHeight = maturity + 15;
 
-                // Wait for block repo for block sync to work
-                TestHelper.WaitLoop(() => TestHelper.IsNodeSynced(stratisSender));
```

Comment:
Not needed?

Reply:
Nope this is done in `TestHelper.MineBlocks`

Reply:
ah yes